### PR TITLE
Merge1.0.1

### PR DESCRIPTION
**Pull request by demonodojo**

### CODE REVIEW

File: sinfonier-backend-api/tests/data/topologies.json [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+            "position": [
+              85,
+              217
+            ]
+          },
+          "value": {
+          }
+        }
+      ],
+      "properties": {
```
Comment (fandrod) [inline sinfonier-backend-api/tests/data/topologies.json]: With last changes on "develop/1.1.0", this attribute must be called "stormProperties" or "topologyProperties"

File: sinfonier-backend-api/tests/data/topologies.json [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+            ],
+            "valuelist": [
+              "https://jsonplaceholder.typicode.com/posts/1"
+            ],
+            "frequency": 10,
+            "delay": 5
+          }
+        }
+      ],
+      "properties": {
```
Comment (fandrod) [inline sinfonier-backend-api/tests/data/topologies.json]: With last changes on "develop/1.1.0", this attribute must be called "stormProperties" or "topologyProperties"

File: sinfonier-web/app/controllers/Modules.java [hunk 3]
```diff
@@ -22,6 +22,7 @@
 import exceptions.SinfonierException;
 import models.module.Container;
 import models.module.Module;
+import models.module.ModulesContainer;
```
Comment (fandrod) [inline sinfonier-web/app/controllers/Modules.java]: Duplicated import

File: sinfonier-web/test/data/topologies.json [hunk 4]
```diff
@@ -138,5 +138,185 @@
...
+            "xtype": "WireIt.FormContainer",
+            "position": [
+              85,
+              217
+            ]
+          },
+          "value": {}
+        }
+      ],
+      "properties": {}
```
Comment (fandrod) [inline sinfonier-web/test/data/topologies.json]: With last changes on "develop/1.1.0", this attribute must be called "stormProperties" or "topologyProperties"

File: sinfonier-web/test/data/topologies.json [hunk 5]
```diff
@@ -138,5 +138,185 @@
...
+            ],
+            "valuelist": [
+              "https://jsonplaceholder.typicode.com/posts/1"
+            ],
+            "frequency": 10,
+            "delay": 5
+          }
+        }
+      ],
+      "properties": {}
```
Comment (fandrod) [inline sinfonier-web/test/data/topologies.json]: With last changes on "develop/1.1.0", this attribute must be called "stormProperties" or "topologyProperties"

File: sinfonier-web/app/views/Drawers/index.html [hunk 6]
```diff
@@ -221,30 +223,72 @@
...
       });
     }, 1000);
-  });
-    
-  $("#changeToen,#changeToes").each(function(){
-    $(this).submit(function(event) {
-      adviseTopologyNotSaved(event);
+   });
+  
+  function regenerateModulesDD() {
```
Comment (fandrod) [inline sinfonier-web/app/views/Drawers/index.html]: This function was moved to drawer.js file

File: sinfonier-web/app/views/Drawers/index.html [hunk 7]
```diff
@@ -221,30 +223,72 @@
...
   });
   
-  $("a[id^='menu-a'],a[id^='header-a'],#Logout").each(function(){
+  $("a[id^='menu-a'],a[id^='header-a']").each(function(){
     $(this).click(function(event) {
       adviseTopologyNotSaved(event);
     });
   });
   
+  function adviseTopologyNotSaved(event, form) {
```
Comment (fandrod) [inline sinfonier-web/app/views/Drawers/index.html]: This function was moved to drawer.js file, but with an old implementation

**Review by fandrod — APPROVED** [review]

**Review by fandrod — APPROVED** [review]

**Review by amatosc — CHANGES_REQUESTED** [review]
check the test of api please.

**Review by amatosc — APPROVED** [review]

**Pull request merged**